### PR TITLE
Feature: Paired t-test table visualization

### PR DESCRIPTION
**Pull request by Mogball**
Continuation of https://github.com/apache/incubator-superset/pull/3006

Paired t-test GIF

Features
- Displays p-values and lift values from Student's t-test between any number of time series
- Click a row on the table to select the control value
- Multiple selected metrics give one table each
- Table columns are sortable
- Control row to adjust significance level (default 0.05) and decimal precisions
- Displays statistically significant time series to the selected control

Differences to #3006 
- Does not contain a line graph
- Supports multiple metrics
- Each group name is display in its own column

### CODE REVIEW

**Comment by Mogball** [comment]
Will add unit tests soon

**Comment by Mogball** [comment]
@mistercrunch Where should I put unit tests for this viz?

File: superset/assets/visualizations/paired_ttest.jsx [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+      liftValues: [],
+      control: 0,
+    };
+  }
+
+  componentWillMount() {
+    this.computeTTest(this.state.control);
+  }
+
+  getLiftStatus(row) {
```
Comment (joshwalters) [inline superset/assets/visualizations/paired_ttest.jsx]: Could you add some code comments on some of these functions? I get the general idea of what the functions do, but some comments might help with future clarity.

**Review by joshwalters — COMMENTED** [review]
70% code coverage, nice!

Edit: Actually just over the hump of 69.5%, but close enough. 😄

**Comment by Mogball** [comment]
@mistercrunch The travis CI building doesn't seem to be running on this PR

**Comment by kkalyan** [comment]
This should be very useful, thanks @Mogball!

**Pull request merged**